packages/dds/tree/src/feature-libraries/modular-schema/modularChangeCodecs.ts

```diff
@@ -265,6 +280,20 @@ export function makeV0Codec(
 		return map;
 	}
 
+	function decodeBuilds(
+		encoded: EncodedBuilds | undefined,
+		context: ChangeEncodingContext,
+	): ModularChangeset["builds"] {
+		return decodeDetachedNodes(encoded, context);
+	}
+
+	function decodeRefreshers(
+		encoded: EncodedBuilds | undefined,
+		context: ChangeEncodingContext,
+	): ModularChangeset["refreshers"] {
+		return decodeDetachedNodes(encoded, context);
+	}
```

Comment:
Same question: why not call `decodeDetachedNodes` directly instead of having a function that adds a level of indirection (let alone two functions)?